CMakeLists.txt

```diff
@@ -1,34 +1,34 @@
 
 # Options
 
-option(BUILD_SHARED_LIBS "Build shared libs" ON)
-option(BUILD_STATIC_TEST_BINS "Build statically linked test executables" OFF)
-option(ENABLE_LTO "Enable LTO on GCC or ThinLTO on clang" OFF)
-option(BUILD_LIBM "libsleef will be built." ON)
-option(BUILD_DFT "libsleefdft will be built." OFF)
-option(BUILD_QUAD "libsleefquad will be built." OFF)
-option(BUILD_GNUABI_LIBS "libsleefgnuabi will be built." ON)
-option(BUILD_SCALAR_LIB "libsleefscalar will be built." OFF)
-option(BUILD_TESTS "Tests will be built." ON)
-option(BUILD_INLINE_HEADERS "Build header for inlining whole SLEEF functions" OFF)
+option(SLEEF_BUILD_SHARED_LIBS "Build shared libs" ON)
```

Comment:
We think this option should be removed altogether and we should keep using the standard cmake variable BUILD_SHARED_LIBS in the rest of the build system. This way we ensure consistency with projects using SLEEF.
Remember that even if turned off libsleefgnuabi will be a shared library.

Reply:
if not set SLEEF_BUILD_SHARED_LIBS , and then use BUILD_SHARED_LIBS ? let SLEEF_BUILD_SHARED_LIBS overwrite BUILD_SHARED_LIBS?